tests/commands/test_bump_command.py

```diff
@@ -66,6 +66,24 @@ def test_bump_minor_increment_annotated(commit_msg, mocker):
     cmd_res = cmd.run('git for-each-ref refs/tags --format "%(objecttype):%(refname)"')
     assert tag_exists is True and "tag:refs/tags/0.2.0\n" in cmd_res.out
 
+    _is_signed = git.is_signed_tag("0.2.0")
+    assert _is_signed is False
+
+
+@pytest.mark.parametrize("commit_msg", ("feat: new file", "feat(user): new file"))
+@pytest.mark.usefixtures("tmp_commitizen_project_with_gpg")
+def test_bump_minor_increment_signed(commit_msg, mocker):
+    create_file_and_commit(commit_msg)
+    testargs = ["cz", "bump", "--yes", "--signed-tag"]
```

Comment:
Should we use something like in git commit instead? I think they use `-s` and some others, @Lee-W ?

Reply:
@Woile I'm not sure I get your point. But using `cz bump ` to test here should be fine

Reply:
I was asking about the `flag` name, should we use something like `git commit -s`? In the current implementation is using `--signed-tag` or `-st`

Reply:
I put `--signed-tag` and `-st` to mimic what's done for annotated tag.

No problem to change to `-s` like git for the short name, let me know.

Is long name `--signed-tag` ok?

Reply:
Makes sense, I'm not really sure what would be correct. 

For example, do you think it would be possible to sign an anottated tag and a lightweight tag? Could a user do:

```
cz bump --anotated-tag -s
```
and also
```
cz bump -s
```

Reply:
A tag can be lightweight, or annotated, or signed.

This is why into tag method the `_opt` is not appended but substituted (cannot link the row, so paste here):
```python
    if annotated:
         _opt = f"-a {tag} -m"
     if signed:
         _opt = f"-s {tag} -m"
```

I think git cannot accept both arguments, but I'm not sure.

Reply:
looking at git documentation, there are:
- `-s` and `--sign` for [git tag](https://git-scm.com/docs/git-tag#Documentation/git-tag.txt--s)
- `-S` `--gpg-sign` for [git commit](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--Sltkeyidgt)

If commitizen should use same git options' name, I think also commit command should be aligned.

Let me know if you agree and I'll align both commands.

Reply:
yes, i agree